src/python/zquantum/qaoa/farhi_ansatz_test.py

```diff
@@ -1,8 +1,9 @@
 from zquantum.core.interfaces.ansatz_test import AnsatzTests
 from zquantum.core.circuit import Circuit, Gate, Qubit
 from zquantum.core.utils import compare_unitary
+from zquantum.core.openfermion import change_operator_type
 from .farhi_ansatz import QAOAFarhiAnsatz
-from openfermion import QubitOperator
+from openfermion import QubitOperator, IsingOperator
 import unittest
```

Comment:
I would say that now would be a good time to refactor these tests to use pytest unless there's a good reason not to.

Reply:
I want to do a separate PR, because it requires changes in core first and then a lot of refactoring in all our **components**.